clients/admin-ui/src/features/common/DeleteConfirmation.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  SimpleGrid,
+} from "@fidesui/react";
+import { ReactNode } from "react";
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+  onDelete: () => void;
+  title: string;
+  message: ReactNode;
+}
+const DeleteConfirmation = ({
```

Comment:
This component isn't really "delete" specific, seeing as the message is nice and generic. It could be `ConfirmationModal` with an `onConfirm` callback.

Reply:
oooh good point! will change 👍